Update README.md (#175)
sveltejs/template: README.md

By default, the server will only respond to requests from localhost. To allow connections from other computers, edit the `sirv` commands in package.json to include the option `--host 0.0.0.0`.

> If you're using [Visual Studio Code](https://code.visualstudio.com/) you can install the official extension [Svelte for VS Code](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode), and if you are using other editors you may need to install a plugin in order to get syntax highlighting and code hints, etc.

Reviewer: ```suggestion
If you're using [Visual Studio Code](https://code.visualstudio.com/) we recommend installing the official extension [Svelte for VS Code](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode). If you are using other editors you may need to install a plugin in order to get syntax highlighting and intellisense.
```

Author: Ah now that is the right tone 👍
